config-model/src/main/java/com/yahoo/vespa/model/content/cluster/DomTuningDispatchBuilder.java

```diff
@@ -25,12 +25,11 @@ public static DispatchTuning build(ModelElement contentXml, DeployLogger logger)
         builder.setMaxHitsPerPartition(dispatchElement.childAsInteger("max-hits-per-partition"));
         builder.setTopKProbability(dispatchElement.childAsDouble("top-k-probability"));
         builder.setDispatchPolicy(dispatchElement.childAsString("dispatch-policy"));
-        builder.setMinGroupCoverage(dispatchElement.childAsDouble("min-group-coverage"));
         builder.setMinActiveDocsCoverage(dispatchElement.childAsDouble("min-active-docs-coverage"));
 
         if (dispatchElement.child("use-local-node") != null)
```

Comment:
You should add a similar warning if 'min-group-coverage' has been configured.

Reply:
Good point; done